package.json

```diff
@@ -145,7 +145,7 @@
     "@emotion/react": "11.11.1",
     "@guardian/ab-core": "2.0.0",
     "@guardian/ab-react": "2.0.1",
-    "@guardian/commercial": "23.7.4",
+    "@guardian/commercial": "^23.7.5",
```

Comment:
what does the extra `^` thing do?

Reply:
according to google it lets it automatically update the minor and patch version.  But isn't that what package.json is supposed to prevent?

Reply:
if you don't use the caret then yes. If you have confidence that minor version bumps won't break anything then you can add the caret and let npm/yarn install the latest in the line of minor versions